implementations/rust/ockam/ockam_command/src/enroll/command.rs

```diff
@@ -50,21 +60,62 @@ impl EnrollCommand {
         initialize_identity_if_default(&opts, &self.identity);
         node_rpc(rpc, (opts, self));
     }
+
+    pub fn is_running() -> bool {
+        if let Ok(path) = CliState::default_dir() {
+            return Path::new(&path.join("enroll_touch_file")).exists();
+        }
+        false
+    }
 }
 
 async fn rpc(ctx: Context, (opts, cmd): (CommandGlobalOpts, EnrollCommand)) -> miette::Result<()> {
     run_impl(&ctx, opts, cmd).await
 }
 
+fn set_signal_handler(terminal: Terminal<TerminalStream<Term>>) {
+    let confirmation = Arc::new(AtomicBool::new(false));
+    ctrlc::set_handler(move || {
+        if confirmation.load(std::sync::atomic::Ordering::Relaxed) {
```

Comment:
Consider importing `std::sync::atomic::Ordering::Relaxed` to reduce visual noise.